project_leverage/models/account_analytic_line.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2020 Open Source Integrators
+# Copyright (C) 2020 Serpent Consulting Services
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    project_user_id = fields.Many2one('res.users',
+                                      related='task_id.project_id.user_id',
+                                      string='Project Manager',
+                                      store=True,
+                                      readonly=True)
```

Comment:
README says this supports Project manager changes, but I'm not sure.
When the PM is changed, the related Analytic Lines will all be recomputed and updated.
I think we need a non computed field, and logic on create/write to set this value.